From a0a93d99939de297863f7060e78e6a0ce7a271a7 Mon Sep 17 00:00:00 2001
From: Sergei Golubchik <sergii@pisem.net>
Date: Tue, 18 Nov 2014 22:26:04 +0100
Subject: [PATCH] MDEV-4399 mysql_secure_installation reports error in
 find_mysql_client

1. remove find_mysql_client (from a bad merge)
2. use $mysql_command
---
 scripts/mysql_secure_installation.sh | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/scripts/mysql_secure_installation.sh b/scripts/mysql_secure_installation.sh
index 9e9bce9..8eca327 100644
--- a/scripts/mysql_secure_installation.sh
+++ b/scripts/mysql_secure_installation.sh
@@ -182,7 +182,7 @@ else
 fi
 
 mysql_command=`find_in_basedir mysql $bindir`
-if test -z "$print_defaults"
+if test -z "$mysql_command"
 then
   cannot_find_file mysql $bindir
   exit 1
@@ -204,7 +204,7 @@ prepare() {
 do_query() {
     echo "$1" >$command
     #sed 's,^,> ,' < $command  # Debugging
-    $bindir/mysql --defaults-file=$config <$command
+    $mysql_command --defaults-file=$config <$command
     return $?
 }
 
@@ -376,7 +376,6 @@ clean_and_exit() {
 # The actual script starts here
 
 prepare
-find_mysql_client
 set_echo_compat
 
 echo
-- 
2.5.5

